VoodooI2C/VoodooI2C/VoodooI2CDevice/VoodooI2CDeviceNub.cpp

```diff
@@ -19,6 +19,8 @@
 OSDefineMetaClassAndStructors(VoodooI2CDeviceNub, IOService);
 
 bool VoodooI2CDeviceNub::attach(IOService* provider, IOService* child) {
+    char const *interruptMode = "Polling";
```

Comment:
Can you switch to the more-common `const char *` prefix, to keep it the same as the rest of the code?